float64 X_Axis
float64 Y_Axis
float64 Z_Axis
float64 x
float64 y
float64 z
float64 w